src/backend/distributed/commands/database.c

```diff
@@ -293,7 +293,9 @@ PreprocessAlterDatabaseStmt(Node *node, const char *queryString,
 		return NIL;
 	}
 
-	EnsureCoordinator();
+
+	EnsurePropagationToCoordinator();
+	EnsureAllObjectDependenciesExistOnAllNodes(list_make1(dbAddress));
```

Comment:
If we have some bugs caused by not calling EnsureAllObjectDependenciesExistOnAllNodes in the functions defined in this file, I think we should fix them separately.

Reply:
So can we remove them from the PR?

Reply:
So can we remove them from the PR?

Reply:
You pointed out that we should not fix them in this PR. And I approved it and assumed that there is nothing to do with this comment. 
I need more clarification here on what needs to be done here. Are you asking if we can remove selected changes? If so, this block is necessary to propagate non-main db operations, therefore we need to keep it